test/util/KeyEvents.js

```diff
@@ -1,8 +1,9 @@
-export function createKeyEvent(element, code, ctrlKey) {
+export function createKeyEvent(element, key, ctrlKey) {
   var e = document.createEvent('Events') || new document.defaultView.CustomEvent('keyEvent');
 
-  e.keyCode = code;
-  e.which = code;
+  e.key = key;
```

Comment:
If we decide to only look into `event.key` we should probably not mock the other properties, right?